Sources/Core/JavaModelRenderer.swift

```diff
@@ -417,8 +432,10 @@ public struct JavaModelRenderer: JavaFileRenderer {
             default: return nil
             }
         }
-
-        let additionalImports = propertyTypeImports + uriType.imports + (unknownPropertyLogging?.imports ?? []) + (decorations.imports ?? [])
```

Comment:
I was getting an error "The compiler is unable to type-check this expression in reasonable time; try breaking up the expression into distinct sub-expressions" if I tried to keep adding more elements to `additionalImports` in the sma line.